### ISSUE TITLE
[COGNITION]: VAOS Scheduler - Contact Info page needs an H1 and active focus management

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#2862]
## Description

The Contact Info page should have an H1 that can identify it to users and give us a hook to set focus on page load. This is a high priority item because of its relationship to #2846 and improvement in the user experience.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Related Issues
* #2846 

## Acceptance Criteria
* As a screen reader user, I want to hear the `<h1>` read aloud when the page loads
* As this user, I want the benefit of being oriented on the page when the `<h1>` takes focus on load

## WCAG or Vendor Guidance (optional)

* [Focus Order: Understanding SC 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html)

## Screenshots or Trace Logs

![Screen Shot 2019-10-29 at 9 48 03 AM](https://user-images.githubusercontent.com/934879/67790836-6e7e4580-fa44-11e9-8e81-68d3709007e3.png)
